File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.js
var reverseWords = function (s) {
    function formatStr(arr) {
        let str = '';
        let length = arr.length;
        for (let i = 0; i < length; i++) {
            if (i === length - 1)
                str += arr[i];
            else
                str += arr[i] + " ";

        }

        return str
    }
    s = s.split(' ').filter(e => e !== '').reverse();
    return formatStr(s);
};

File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.cs
public class Solution {
    public string ReverseWords(string s) {
        string[] words = s.Split(' ', StringSplitOptions.RemoveEmptyEntries);
        Array.Reverse(words);
        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < words.Length; i++)
        {
            sb.Append(words[i]);
            if(i < words.Length - 1) sb.Append(' ');
        }
        return sb.ToString();
    }
}